spec/controllers/tags_controller_spec.rb

```diff
@@ -533,5 +533,18 @@
         end
       end
     end
+
+    it "can add and remove metatags at the same time" do
+      old_metatag =  create(:canonical_freeform)
+      new_metatag =  create(:canonical_freeform)
+
+      tag = create(:canonical_freeform)
+      MetaTagging.create(meta_tag: old_metatag, sub_tag: tag, direct: true)
```

Comment:
Could this be replaced with the following?
```suggestion
      create(:meta_tagging, meta_tag: old_metatag, sub_tag: tag, direct: true)
``